SetupDataPkg/Plugins/UpdateConfigHdr/UpdateConfigHdr.py

```diff
@@ -23,13 +23,17 @@ def do_pre_build(self, thebuilder):
             "SetupDataPkg", "Test", "Include"
         )
 
-        final_dir = thebuilder.env.GetValue("CONF_AUTOGEN_INCLUDE_PATH", default_generated_path)
+        # we can have a semicolon delimited list of include paths to generate to
+        dirs = thebuilder.env.GetValue("CONF_AUTOGEN_INCLUDE_PATH", default_generated_path).split(";")

```

Comment:
Typically, you want to split by `os.pathsep` which is ":" or ";" depending on the OS. At a bare minimum you should split on ":" or ";" to support the canonical way to separate paths for both windows and linux OS.

Reply:
You also do a lot of splitting down below using just ";" so if you decide to change it here, you'll want to change it below too.

Reply:
Regardless of it being incidental, the standard way when combining paths is to use the OS specific separator. Regardless, you have it well documented so it should not be a problem in the future.